### PR TITLE
(DOCSP-26250): App Services release notes 03-11-2022

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

App Services release notes for Nov. 3, 2022 release.

### Jira

- https://jira.mongodb.org/browse/DOCSP-26250

### Staged Changes (Requires MongoDB Corp SSO)

- [Release Notes](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-26250/release-notes/backend/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst [hunk 1]
```diff
@@ -1,3 +1,13 @@
+.. _backend_20221102:
+
+2 November 2022 Release
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- :ref:`Function context.app.id <context-app>` returns a string (formerly BSON ObjectId).
+- Support :ref:`App Services deployments <deployment-regions>` in the AWS region
+  London (aws-eu-west-2).
+- Added Admin API Endpoint for App metrics.
```
Comment (dacharyc) [inline source/release-notes/changelogs/backend/changelog-backend-2022.rst:9]: Should/can we link to this endpoint? I think it's the one added in #180

**Review by dacharyc — APPROVED** [review]
Overall LGTM - just a question about whether we should link to docs for the third bullet item.

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst (hunk 1 above)
Comment (mongodben) [reply source/release-notes/changelogs/backend/changelog-backend-2022.rst:9]: i didn't link to it b/c it's not a ref. but let me link to ref for Admin API + specify the exact name used there.

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst [hunk 2]
```diff
@@ -1,3 +1,13 @@
+.. _backend_20221102:
+
+2 November 2022 Release
```
Comment (mongodben) [inline source/release-notes/changelogs/backend/changelog-backend-2022.rst:3]:
```suggestion
.. _backend_20221103:

3 November 2022 Release
```

**Pull request merged**